requirements.txt

```diff
@@ -35,3 +35,6 @@ slacker==0.7.0
 # Documentation requirements
 Sphinx==1.3.1
 sphinx-autobuild==0.5.2
+
+# Journalism Workflow requirements
```

Comment:
Left it here to simplify the installation workflow. I think we discussed having a `setup_orchestra.sh` script in the pypi package--when we write that I'd say it's reasonable to give it commandline arguments like `--journalism-workflow` to indicate that those requirements are needed.